common/swagger/swagger.yaml

```diff
@@ -203,6 +288,14 @@ definitions:
         type: array
         items:
           $ref: '#/definitions/Row'
+      featureShapleyContributions:
+        type: object
+        description: >
+          An object with features and shapley values that was requested by the client.
+          It is currently available for transformed features of binomial, regression and multinomial models of mojo2.
+          This field will not be populated if the Shapley values are not available for a model.
+        items:
+          $ref: '#/definitions/ContributionResponse'
```

Comment:
I am a bit confused with the `items`.

Specification says _[...] Describes the type of items in the array._

How does it work with `type: object`?